app/src/main/java/com/cornellappdev/coffee_chats_android/networking/NetworkingUtils.kt

```diff
@@ -120,6 +120,16 @@ suspend fun updateSocialMedia(socialMedia: SocialMedia): ApiResponse<SocialMedia
 suspend fun updateOnboardingStatus(hasOnboarded: Boolean): ApiResponse<OnboardingStatus>? =
     postDataHelper(Endpoint.updateOnboardingStatus(hasOnboarded), OnboardingStatus::class.java)
 
+// FCM TOKEN
+
+suspend fun updateFcmToken(fcmToken: String): ApiResponse<FcmToken>? =
+    postDataHelper(Endpoint.updateFcmToken(fcmToken), FcmToken::class.java)
+
+// MESSAGE NOTIFICATION
+
+suspend fun updateMessage(message: String, recipientId : Int): ApiResponse<OnboardingStatus>? =
```

Comment:
Nit: could you change the name of this function to something like `sendMessageNotification`? The current name makes it seem like we're editing an existing message.